snuba/utils/metrics/backends/abstract.py

```diff
@@ -0,0 +1,24 @@
+from abc import ABC, abstractmethod
+from typing import Optional, Union
+
+from snuba.utils.metrics import Tags
+
+
+class MetricsBackend(ABC):
```

Comment:
This class isn't covered, but it shouldn't need to be since all of the methods are abstract.